yarn.lock

```diff
@@ -10512,11 +10512,16 @@ eth-block-tracker@^4.4.2:
     pify "^3.0.0"
     safe-event-emitter "^1.0.1"
 
-eth-contract-metadata@^1.11.0, eth-contract-metadata@^1.16.0:
+eth-contract-metadata@^1.11.0:
```

Comment:
This could be deduplicated, but it's not important functionally (this is brought in by `@metamask/controllers`, but isn't used by any controllers we use)